package.json

```
@@ -310,7 +310,7 @@
     "proxy-from-env": "1.0.0",
     "proxyquire": "1.8.0",
     "puid": "1.0.7",
-    "puppeteer": "npm:@elastic/puppeteer@5.4.1-patch.1",
+    "puppeteer": "8.0.0",
```

Also, there's a new v9.0.0 out now (8 days ago) - shouldn't we just upgrade to that now that we're at it? (also comes with support for M1 😍)

```suggestion
    "puppeteer": "^9.0.0",
```

---

Puppeteer v9.0.0 is tied to Chromium r869685, which is commit 362f49f9988c8051e610ed5e21047a5876e9915f. 

I ran a test to try to build that commit with our scripts, and it looks like there is a bug in the Chromium source code at that point of history:
```
 > autoninja -C out/headless headless_shell
ninja: Entering directory `out/headless'
[3478/34986] ACTION //third_party/icu:make_data_assembly(//build/toolchain/linux:clang_x64)                                                                  
FAILED: gen/third_party/icu/icudtl_dat.S 
python3 ../../third_party/icu/scripts/make_data_assembly.py ../../third_party/icu/common/icudtl.dat gen/third_party/icu/icudtl_dat.S
  File "../../third_party/icu/scripts/make_data_assembly.py", line 102
    print "Generated " + output_file
          ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print("Generated " + output_file)?
```

It might take some time to figure out how to fix that, or find a compatible commit that does not have the bug. For now, I think we should stick with upgrading Puppeteer to 8.0.0 since it is tied to Chromium r856583 and the build is working for that version.

---

I can spend more time looking into this after this upgrade is merged